### DISCUSSION TITLE
How to get table recognition algorithm named SLANet research paper?

### DISCUSSION THREAD

**Discussion by TapendraBaduwal** [PaddlePaddle/PaddleOCR#8292] [General]
How to get table recognition algorithm named SLANet research paper?

**Comment by CycloneBoy** [comment]
you can read this paper [PP-StructureV2: A Stronger Document Analysis System](https://arxiv.org/abs/2210.05391)

![image](https://user-images.githubusercontent.com/13173452/202147420-89b3846f-9e0d-44b3-afd8-60da1638ba0f.png)

**Reply by TapendraBaduwal** [reply]
@CycloneBoy Thank you
